eZ/Publish/API/Repository/Values/URL/Query/Criterion/Pattern.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace eZ\Publish\API\Repository\Values\URL\Query\Criterion;
+
+/**
+ * Matches URLs which contains the pattern.
+ */
+class Pattern extends Matcher
+{
+    /**
+     * @var string|null
```

Comment:
expand on the format please, be precise on what is supported.

Reply:
Done in 36a6dbbf5dac971d849f1f0efa05adf1a2f54322.